scripts/SLT1-JP/c1002702008.lua

```diff
@@ -0,0 +1,46 @@
+--ヒエラティックヘブンリードラゴンヘリオポリスの領主
+--LUA by Kohana Sonogami
+--
+function c55285840.initial_effect(c)
+	c:EnableReviveLimit()
+	aux.AddXyzProcedure(c,nil,8,2)
+	--negate
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(1002702008,0))
+	e1:SetCategory(CATEGORY_NEGATE+CATEGORY_DESTROY)
+	e1:SetType(EFFECT_TYPE_QUICK_O)
+	e1:SetCode(EVENT_CHAINING)
+	e1:SetProperty(EFFECT_FLAG_DAMAGE_STEP+EFFECT_FLAG_DAMAGE_CAL)
+	e1:SetRange(LOCATION_MZONE)
+	e1:SetCountLimit(1)
+	e1:SetCondition(c1002702008.discon)
+	e1:SetCost(c1002702008.discost)
+	e1:SetTarget(c1002702008.distg)
+	e1:SetOperation(c1002702008.disop)
+	c:RegisterEffect(e1)
+end
+function c1002702008.tfilter(c,tp)
+	return (c:IsLocation(LOCATION_ONFIELD+LOCATION_GRAVE) or c:IsFaceup()) and c:IsControler(tp)
```

Comment:
It also includes the facedown banished cards, suggested change:
`return c:IsLocation(LOCATION_ONFIELD+LOCATION_GRAVE+LOCATION_REMOVED) and c:IsControler(tp)`